src/separations_tests.cc

```diff
@@ -42,7 +42,153 @@ TEST(SeparationsTests, SepMaterial) {
   EXPECT_DOUBLE_EQ(effs[id("Am241")] * mqorig.mass("Am241"), mqsep.mass("Am241"));
   EXPECT_DOUBLE_EQ(0, mqsep.mass("Am242"));
 }
+  
+TEST(SeparationsTests, SeparationEfficiency) {
```

Comment:
This test is sort of implicit in all the other separations tests that already don't throw.  We can probably just remove it.